src/dbnode/storage/namespace/schema.go

```diff
@@ -113,6 +112,19 @@ func (sr *schemaRegistry) Equal(o SchemaRegistry) bool {
 	return true
 }
 
+func (sr *schemaRegistry) Lineage(v SchemaRegistry) bool {
```

Comment:
Can we call this `Extends`? I would expect `Lineage` to return an actual slice or something

Reply:
sure.